pkg/resources/fluentd/config.go

```diff
@@ -33,8 +33,10 @@ var fluentdInputTemplate = `
 # Enable RPC endpoint (this allows to trigger config reload without restart)
 <system>
   rpc_endpoint 127.0.0.1:24444
-  {{- if .LogFormat }}
-  format {{ .LogFormat }}
+  {{- if .LogFormat }}  
+  <log>
+    format {{ .LogFormat }}
+  </log>format {{ .LogFormat }}
```

Comment:
```suggestion
  </log>
```
I feel like this should be removed.